ereolen_support/ereolen_support.form.inc

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/**
+ * @file
+ * Add eReolen support form.
+ */
+
+include_once 'ereolen_support.helper.inc';
+
+/**
+ * Create frontend form.
+ *
+ * @param array $form
+ *   The form to create.
+ * @param array $form_state
+ *   The state of the form.
+ *
+ * @return mixed
+ */
+function ereolen_support_form($form, &$form_state) {
+  global $user;
+  $form['#attributes'] = array(
+    'name' => 'ereolen_support_form',
+    'class' => array('node-webform')
```

Comment:
webform class on a non webform?

Reply:
It is a form on the web, so it is not an incorrect class for this particular object. This class happens to bring with it a lot of styling that we want.